backend/src/zelthy/cli/project_template/project_name/settings.py

```diff
@@ -116,3 +116,15 @@
 STATIC_ROOT = os.path.join(BASE_DIR, "static")
 STATIC_URL = "static/"
 STATICFILES_DIRS += [os.path.join(BASE_DIR, "assets")]
+
+# Axes Lockout
+env = environ.Env(
```

Comment:
No need to initialize it over here, it's initialized above (line no 9)
you can just update your default values in it